src/stacks-hierarchy/RootStack.tsx

```diff
@@ -126,18 +127,24 @@ export const RootStack = () => {
     ];
 
     if (path.includes('details')) {
+      const item: ServiceJourneyDeparture = {
+        serviceJourneyId: params.serviceJourneyId as string,
+        date: (params.date as string) || new Date().toISOString(),
+        serviceDate: params.serviceDate as string,
+        fromStopPosition:
+          typeof params.fromStopPosition === 'string'
+            ? parseInt(params.fromStopPosition)
+            : 0,
```

Comment:
@jorelosorio I'm having an issue where the deep link that's built by a set of `URLQueryItem`s in WidgetViewModel added quotation marks to the `fromStopPosition` param, even thought it should be a plain integer (`fromStopPosition="10"` instead of `fromStopPosition=10`).

Running `parseInt` like this works, but do you know if there's a way to fix it in Swift?

Reply:
I can have a look ;)

Reply:
I was looking into the documentation of how the URLQueryItem works but it seems like all the values must be strings, because that is what is sent to the app when it opens, so even if we send an integer while building the URL, it might at the end send a string with the `scheme://widget/......`

Reply:
Thanks for looking into it! I had another look now, and I believe `URLQueryItem` is doing it correctly, and sending `fromStopPosition=10`. What confused me was that the param type in react native is `string | number | boolean | ...`, but the data always seem to be `string` 🫠 

I added a util now to clean the code in RootStack up a bit.